code/game/objects/items/weapons/clown_items.dm

```diff
@@ -93,7 +93,7 @@
 	..()
 
 /obj/item/weapon/soap/nanotrasen
-	desc = "A NanoTrasen-brand bar of soap. Smells of plasma."
+	desc = "A NeoTheology-brand bar of soap. Smells of biomatter."
```

Comment:
Change "biomatter" to "vomit" to fit in with its taste description.

Reply:
![PC4j8HIaUcA2](https://user-images.githubusercontent.com/9161564/69790730-0d2ccc00-11d4-11ea-9c38-b0543ee6b67a.png)